sputnikdao2/src/proposals.rs

```diff
@@ -551,4 +559,18 @@ impl Contract {
             log!("Memo: {}", memo);
         }
     }
+
+    /// Actions on the list of proposals
```

Comment:
`Act on given proposals, sequentially. The execution is interrupted and reverted if any of the proposal actions causes panic.`